<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkHistogramVisualizationControls</class>
 <widget class="QWidget" name="QmitkHistogramVisualizationControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidgetPlot">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabWidgetPlotPage1">
      <attribute name="title">
       <string>Options</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QCheckBox" name="checkBoxShowSubchart">
         <property name="text">
          <string>Show Subchart</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxUseDefaultNBins">
         <property name="text">
          <string>Use default #bins</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="FrameBinSize">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="1" column="0">
           <widget class="QLabel" name="labelNBins">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string># bins:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QSpinBox" name="spinBoxNBins">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Press enter to recalculate statistics with new bin size.</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabWidgetPlotPage2">
      <attribute name="title">
       <string>Custom Zoom</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="checkBoxViewMinMax">
         <property name="text">
          <string>Zoom to min/max values</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frameViewMinMaxValues">
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="1">
           <widget class="QLabel" name="labelMinValue">
            <property name="text">
             <string>Minimum value</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxMinValue"/>
          </item>
          <item row="2" column="0">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxMaxValue"/>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="labelMaxValue">
            <property name="text">
             <string>Maximum value</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QmitkChartWidget" name="chartWidget" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="widgetClipboard" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="buttonCopyHistogramToClipboard">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Copy to Clipboard</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkChartWidget</class>
   <extends>QWidget</extends>
   <header location="global">QmitkChartWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
